editor/block-settings-menu/unknown-converter.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { IconButton } from '@wordpress/components';
+import { getUnknownTypeHandlerName, pasteHandler } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { getBlock } from '../selectors';
+import { replaceBlocks } from '../actions';
+
+export function UnknownConverter( { block, convertToBlocks } ) {
+	if ( ! block || getUnknownTypeHandlerName() !== block.name ) {
+		return null;
+	}
+
+	return (
+		<IconButton
+			className="editor-block-settings-menu__control"
+			onClick={ () => convertToBlocks( block ) }
+			icon="screenoptions"
+		>
+			{ __( 'Convert to blocks' ) }
+		</IconButton>
+	);
+}
+
+export default connect(
+	( state, { uid } ) => ( {
+		block: getBlock( state, uid ),
+	} ),
+	( dispatch, { uid } ) => ( {
+		convertToBlocks( block ) {
+			dispatch( replaceBlocks( [ uid ], pasteHandler( {
+				HTML: block.attributes.content,
```

Comment:
Although we also set it directly in the paste module, we might want to register this as well with the API. See https://github.com/WordPress/gutenberg/blob/1b444cb91ceb4721a3bcc2b3bda1f1d9c4cc1f17/blocks/api/paste/index.js#L109

Atm we require the attribute to be `content` implicitly.